src/codeu/chat/client/core/Context.java

```diff
@@ -43,7 +43,7 @@ public UserContext create(String name) {
   }
 
   public ServerInfo getInfo() {
-    return view.getInfo();
+    return ((View)view).getInfo();
```

Comment:
Instead of casting to View here, I'd recommend changing the variable declaration above to be of type View instead of BasicView - we're never expecting it to hold anything besides a client.core.View.